cli/search/search_test.go

```diff
@@ -5,12 +5,12 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-	"github.com/tarantool/tt/cli/util"
 	"github.com/tarantool/tt/cli/version"
 )
 
 type getVersionsInputValue struct {
-	data string
+	data   *map[string]interface{}
```

Comment:
Map is a pointer wrapper type, so there is no point in using pointers to maps.

Reply:
https://golangbyexample.com/allowed-key-and-value-types-golang/

Reply:
Fixed